docs/howto/without_active_record.md

```diff
@@ -0,0 +1,91 @@
+# How to use ActiveModelSerializers without ActiveRecord
+
+Every model that is to be serialized must implement the following methods
```

Comment:
this entire page can be boiled down to

```ruby
class MyModel
  include ActiveModel::Model
  include ActiveModel::Serialization
end
```

or otherwise passes Lint

```ruby
class MyModelSerializer < ActiveModel::Serializer
  attribute :potato
  def potato; 'potAHto'; end
end
```

and that's it.

Now, outside of the controller the rules are **exactly** the same as for records.

```ruby
ActiveModel::SerializableResource.new(MyModel.new(potato: "potAHto'), adapter: :json)
```

I'd rather just add to the front page readme that AMS requires the thing-to-be-rendered (resource) to pass the Lint, see documentation there. ActiveRecord::Base records pass the lint, as do objects including ActiveModel::Serialization